### ISSUE TITLE
loss functions during pretraining

### ISSUE THREAD

**Issue by mkhoshle** [Arthur151/ROMP#427]
Hi @Arthur151 ,

May I ask what loss functions you have used during your pretraining stage?

Thanks,

**Comment by Arthur151** [comment]
Yes, during pretraining, I used 
1. detection loss to train center map
https://github.com/Arthur151/ROMP/blob/12a624f00be92d1f7195f24106d151d2f26961f4/romp/lib/loss_funcs/calc_loss.py#L39
2. keypoints 2D loss like this
https://github.com/Arthur151/ROMP/blob/12a624f00be92d1f7195f24106d151d2f26961f4/romp/lib/loss_funcs/calc_loss.py#L90
but to learn 2D pose estimation with a separate head.
